portal-frontend/src/app/services/application/application.service.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { firstValueFrom } from 'rxjs';
+import { environment } from '../../../environments/environment';
+import { ApplicationDto, CreateApplicationDto } from './application.dto';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ApplicationService {
+  private serviceUrl = `${environment.apiUrl}/application`;
+
+  constructor(private httpClient: HttpClient) {}
+
+  async createPendingApplication(createDto: CreateApplicationDto) {
+    let formData: FormData = new FormData();
+    formData.append('applicant', createDto.applicant);
+    formData.append('localGovernment', createDto.localGovernmentUuid);
+    if (createDto.documents.length > 0) {
+      formData.append('certificateOfTitle', createDto.documents[0], createDto.documents[0].name);
+    }
+
+    try {
+      await firstValueFrom(this.httpClient.post<ApplicationDto>(`${this.serviceUrl}`, formData));
+      //this.toastService.showSuccessToast('Decision created');
```

Comment:
Is this a placeholder or something that needs to be deleted?

Reply:
I'm not sure, tickets don't mention toasts but I have a feeling they will be wanted.